### DISCUSSION TITLE
Achievement Sub-folders

### DISCUSSION THREAD

**Discussion by coolscribble** [RetroAchievements/RAWeb#745] [Ideas]
Hi, I would love to have some better-organized groups like sometimes is in achievement news: 
level complete, unharmed etc folders

I have in mind something like this: 

![image](https://user-images.githubusercontent.com/32706333/45910707-a12d2780-bdc8-11e8-857a-97d9d5c08380.png)

**Comment by kdecks** [comment]
Achievement folders could be a web only issue and handled in the manage achievements dev tool.

**Comment by coolscribble** [comment]
yeah I know, I don't think it will be hard but it can be more clear

**Comment by Jamiras** [comment]
similar suggestion was posted in the toolkit repo: https://github.com/RetroAchievements/RAIntegration/issues/17

**Comment by coolscribble** [comment]
> 
> 
> similar suggestion was posted in the toolkit repo: [RetroAchievements/RAIntegration#17](https://github.com/RetroAchievements/RAIntegration/issues/17)

I was looking at that, but I didn't know if I may post it there or make new, because it's kind of different

**Comment by meleu** [comment]
Seems to be related to (and better addressed by) #746
